deploy/k8s/chart/Chart.yaml

```diff
@@ -7,9 +7,9 @@ version: 0.1.0
 appVersion: "1.16.0"
 
 dependencies:
-- name: guac
-  version: "0.1.x"
-  repository: "https://kusaridev.github.io/helm-charts"
+- name: nats
```

Comment:
I thought we got rid of nats. What am I misremembering?

Reply:
Nope. At least the Bombastic exporter deployment still indicates a desire to have nats.

We used to get it transitively from the dependency upon the kusari guac charts.